index.css

```diff
@@ -563,3 +561,171 @@ footer {
   padding: 0;
   gap: 0;
 }
+
+/* media query for desktop view styles start here... */
+@media (min-width: 768px) {
+  /* query for header section */
+  header {
+    height: 72px;
+  }
+
+  .logo {
+    font-family: Inter, serif;
+    font-size: 20px;
+    font-weight: 700;
+    line-height: 20px;
+  }
+
+  .menu-icon {
+    display: none;
+  }
+
+  .nav-links {
+    display: flex;
+    justify-content: space-between;
+    gap: 20px;
+  }
+
+  .nav-links > * {
+    text-decoration: none;
+    font-family: Inter, sans-serif;
+    font-size: 15px;
+    font-weight: 600;
```

Comment:
- [ ] From the project requirements, you should stick to the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0), But as you can see there are some differences between the [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122&t=Tt1vCN6lfPrkflxz-0) and your design on desktop view.
  - The `I am always interested ...` should align left not center
  - On desktop view, the contact form has four fields, the `firstname`, `lastname`, `email`, and `message`
  - the `margin-top` of the contact form section should be the same as it is in the Figma design

| Your Design |Figma Design |
| :---:   | :---: | 
| ![Screenshot from 2022-12-14 13-00-41](https://user-images.githubusercontent.com/92303891/207601827-95b28b68-22da-4915-b515-1d493a776c5c.png)|![Screenshot from 2022-12-14 12-59-23](https://user-images.githubusercontent.com/92303891/207601621-bb8dd568-6da8-4232-aec6-13da0ccc736a.png) |

Reply:
thanks!